kuma/settings/common.py

```diff
@@ -886,8 +886,8 @@ def pipeline_one_scss(slug, **kwargs):
             # placed in the kuma/javascript/dist/ directory.
             "react.js",
         ),
-        "output_filename": "build/js/react-main.js",
-        "extra_context": {"defer": True},
+        "output_filename": "build/js/banners.js",
```

Comment:
What?!

Reply:
Ugh yeah, somehow the merge and conflicts just played havoc with this file. Let me sort this one out as well.

Reply:
Sorted.